ocaml/idl/datamodel.ml

```diff
@@ -5634,6 +5633,8 @@ let pif =
       field ~in_oss_since:None ~ty:(Set(Ref _bond)) ~in_product_since:rel_miami ~qualifier:DynamicRO "bond_master_of" "Indicates this PIF represents the results of a bond";
       field ~in_oss_since:None ~ty:(Ref _vlan) ~in_product_since:rel_miami ~qualifier:DynamicRO "VLAN_master_of" "Indicates wich VLAN this interface receives untagged traffic from" ~default_value:(Some (VRef ""));
       field ~in_oss_since:None ~ty:(Set(Ref _vlan)) ~in_product_since:rel_miami ~qualifier:DynamicRO "VLAN_slave_of" "Indicates which VLANs this interface transmits tagged traffic to";
+      field ~in_oss_since:None ~ty:(Ref _net_sriov) ~in_product_since:rel_jura ~qualifier:DynamicRO "sriov_master_of" "Indicates which net_sriov this interface is physical of" ~default_value:(Some (VRef ""));
```

Comment:
The documentation strings need to be revised for grammar and clarity. The names indicate this is about master/slave but the documentation talks about physical/logical. 
`

Reply:
Yes, we have refined the design to update the field name to `sriov_physical_pif_of` and `sriov_logical_pif_of`.